dts/arm/nxp/nxp_k6x.dtsi

```diff
@@ -133,6 +133,8 @@
 			compatible = "nxp,kinetis-sim";
 			reg = <0x40047000 0x1060>;
 			label = "SIM";
+			pllfll-select = <KINETIS_SIM_PLLFLLSEL_MCGFLLCLK>;
```

Comment:
@henrikbrixandersen @MaureenHelm In the ke1xf dts, these are not included but can be found in the board dts file. I think these "default values" would be good to have, what do you think?

Reply:
These are, as far as I can tell, board specific. By not putting in a default you force the board definition authors to consider this.

Reply:
so far I have only made the DT changes in k6x/soc.c, does it make sense to make the dts entries "required: false" and then make it up to the author of the board definitions to include them if supported?

Reply:
What would the driver default to, then? You would just be moving the problem :)

Reply:
The ke18f soc gets these clock settings from kinetis_scg but fails because the sim entries are missing. Maybe the entries should be false and have `#if DT_NODE_HAS_PROP` in the soc files?

Reply:
I don't see the KE18F SoC failing in this PR? It shouldn't since it uses a different SIM devicetree binding.

Reply:
That was an error on my part. I have found that the kw4xz soc does not have pllfll as part of osc_config_t. It makes it weird that pllfll-select is required for that platform.